src/Microsoft.DotNet.ProjectModel/ProjectContext.cs

```diff
@@ -154,39 +154,6 @@ public static IEnumerable<ProjectContext> CreateContextForEachTarget(string proj
                         .BuildAllTargets();
         }
 
-
-        /// <summary>
-        /// Creates a project context based on existing context but using runtime target
-        /// </summary>
-        /// <param name="context"></param>
-        /// <param name="runtimeIdentifiers"></param>
-        /// <returns></returns>
-
-        public ProjectContext CreateRuntimeContext(IEnumerable<string> runtimeIdentifiers)
```

Comment:
I'd still make this available from the project context. Seems more intuitive to me. Maybe it could delegate to the workspace context?  This brings the question, should a ProjectContext know about its WorkspaceContext?